plugins/event_source/kafka.py

```diff
@@ -42,6 +51,17 @@ async def main(queue: asyncio.Queue, args: Dict[str, Any]):
         max_poll_records=1,
         auto_offset_reset=offset,
     )
+
+    if not cafile or not certfile or not keyfile or not password:
```

Comment:
What is the minimal requirement to run in SSL mode? Should the `if` condition be
```
if cafile or certfile or keyfile or password:
```

Reply:
right, `cafile` is required for SSL mode, while `certfile`, `keyfile` and `password` are optional.

Reply:
if `catfile` is required but others are optional, then the condition should be:
```
   if catfile:
```

Reply:
done.